docs/pages/config/database/drivers.mdx

```diff
@@ -141,11 +167,24 @@ export default createConfig({
 </Tab>
 </Tabs>
 
-Example of Redis config with eviction policy:
-```conf
-maxmemory 2000mb
-maxmemory-policy allkeys-lru
-```
+#### Configuration Notes
+
+- When using URI, these parameters are included in the URI string:
+  - `redis://[username]:[password]@[host][:port][/database]`
+  - Database is optional and defaults to 0
+
+- These parameters must be configured separately, even when using URI:
+  - Connection pool size
+  - TLS configuration
+  - Timeout settings (initTimeout, getTimeout, setTimeout)
+
+- If both URI and individual parameters are provided, the URI takes precedence for connection details, but explicit timeouts and connection pool size from the config will still be used.
```

Comment:
imo if a URI is provided, the other config should be ignored since everything should be configurable directly in the URI